Blockzilla/IntroViewController.swift

```diff
@@ -133,7 +213,24 @@ extension IntroViewController: ScrollViewControllerDelegate {
     }
 }
 
-class ScrollViewController: UIPageViewController {
+class ScrollViewController: UIPageViewController, PageControlDelegate {
+    
+    @objc func incrementPage() {
```

Comment:
Confirm: I noticed @objc here but not on decrement, is ok? (just thumb up this comment when you check it)